### PR TITLE
AWS Demo Server

### PR DESCRIPTION
**Pull request by rafaelbernard**
Configurations needed for AWS Demo Server

### CODE REVIEW

File: docker-compose-demoui.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-version: '3.8'
+version: '3.7'
```
Comment (rafaelbernard) [inline docker-compose-demoui.yml:2]: Needed to match EC2 docker version.

File: .env.sample [hunk 2]
```diff
@@ -9,3 +9,6 @@ IRMASERVER_REQUESTORS='{"keyshare_server":{"auth_method":"publickey","key_file":
 
 # Debug
 IRMASERVER_VERBOSE=2
+
+# FOR DEMO SERVER - docker version requires to remove single quote
```
Comment (ehsan-fj) [inline .env.sample]: I didn't get how irma requestor and docker version are related!?

**Review by ehsan-fj — APPROVED** [review]

File: .env.sample (hunk 2 above)
Comment (rafaelbernard) [reply .env.sample]: When docker loads the `.env` file to forward the variables to irma server container, ec2 docker version does not recognize the JSON variable `IRMASERVER_REQUESTORS` when using single quotes and the irma server fails to start.

Comment (ehsan-fj) [reply .env.sample]: I see!!

Comment (rafaelbernard) [inline .env.sample]:
```suggestion
# FOR DEMO SERVER - docker version on ec2 server only reads the JSON value without single quotes
```

**Pull request merged**